src/grimoirelab/core/scheduler/tasks/migrations/0001_initial.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 4.2.14 on 2024-10-29 15:40
+
+from django.db import migrations, models
+import django.db.models.deletion
+import grimoirelab.core.models
+import grimoirelab.core.scheduler.models
+import grimoirelab_toolkit.datetime
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='EventizerTask',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', grimoirelab.core.models.CreationDateTimeField(default=grimoirelab_toolkit.datetime.datetime_utcnow, editable=False)),
+                ('last_modified', grimoirelab.core.models.LastModificationDateTimeField(default=grimoirelab_toolkit.datetime.datetime_utcnow, editable=False)),
+                ('uuid', models.CharField(max_length=191, unique=True)),
+                ('task_type', models.CharField(max_length=128)),
+                ('task_args', models.JSONField(default=None, null=True)),
+                ('status', models.IntegerField(choices=[(1, 'new'), (2, 'enqueued'), (3, 'running'), (4, 'completed'), (5, 'failed'), (6, 'recovery')], default=1)),
+                ('runs', models.PositiveIntegerField(default=0)),
+                ('failures', models.PositiveIntegerField(default=0)),
+                ('last_run', models.DateTimeField(default=None, null=True)),
+                ('scheduled_at', models.DateTimeField(default=None, null=True)),
+                ('job_interval', models.PositiveIntegerField(default=30)),
```

Comment:
When the setup is running, it shows pending changes because the default value from the settings is 86400. Would you prefer to keep it at 30 or update the default interval to 86400?

Reply:
That's right. I should change it. What I had problems was to set a default time for this. I think 30 seconds is too low - for a default value - but 86400 (24 hours) is probably too high.